### ISSUE TITLE
Bulk insert does not work with EntityFrameworkCore.sqlserver 3.1.0

### ISSUE THREAD

**Issue by AJ1000** [borisdj/EFCore.BulkExtensions#268]
Upgraded to EntityFrameworkCore.sqlserver 3.1.0 and BulkInsert stopped working. It works fine is using 3.0.1.

Get the following error:

Use the execution strategy returned by ‘DbContext.Database.CreateExecutionStrategy()’ to execute all the operations in the transaction as a retriable unit.

The code I was running was the following:

DbContext.BulkInsert(orderList, new BulkConfig { SetOutputIdentity = true, PreserveInsertOrder = true });

I am staying with version 3.0.1 for now, but hopefully this is easy to fix.

Thanks.

**Comment by AJ1000** [comment]
I found out why I was getting this error. It was due to me using the EnableRetryOnFailure feature of sqlserverdbcontextoptionsbuilder. I found a solution which is:

var strategy = _catalogContext.Database.CreateExecutionStrategy();

strategy.ExecuteAsync(async () => { 

Using (var transaction = context.Database.BeginTransaction())
{
 // Put bulk insert code in here ....
 transaction.Commit();
} });

The other solution is not to use the EnableRetryOnFailure feature (which is one of the features I want to use).

Hopefully this information helps you in fixing this issue.
